planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/src/scene.cpp

```diff
@@ -602,6 +602,46 @@ void StaticObstacleAvoidanceModule::fillEgoStatus(
     return;
   }
 
+  auto candidate_sl_force_activated = [&](const std::string & direction) {
+    // If statement to avoid unnecessary warning occurring from isForceActivated function
+    if (candidate_uuid_ == uuid_map_.at(direction)) {
+      if (rtc_interface_ptr_map_.at(direction)->isForceActivated(candidate_uuid_)) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  auto registered_sl_force_activated =
+    [&](const std::string & direction, const RegisteredShiftLineArray shift_line_array) {
+      return std::any_of(
+        shift_line_array.begin(), shift_line_array.end(), [&](const auto & shift_line) {
+          return rtc_interface_ptr_map_.at(direction)->isForceActivated(shift_line.uuid);
+        });
+    };
+
+  /**
+   * Check if the candidate avoidance path is force activated
+   */
+  if (isActivated()) {
```

Comment:
Is this line necessary for force approval？

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8094/commits/7eae606aad2090f83cb2af8446fca59a4c00d6bd